azavea/raster-foundry
# Feature/lk/delete dead code

## Overview
Delete old components and pages
Better namespacing and names for components

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

## Testing Instructions

* Verify that moved components still work as expected
 * Verify that none of the stuff I deleted is used anywhere

Closes #1791
